src/OAuth2/OpenID/Storage/UserClaimsInterface.php

```diff
@@ -9,7 +9,8 @@
 interface UserClaimsInterface
 {
     // valid scope values to pass into the user claims API call
-    const VALID_CLAIMS = 'profile email address phone';
+    const VALID_SCOPE_VALUES = 'profile email address phone';
```

Comment:
Unfortunately, we can't remove a class constant w/o breaking BC, so this has to be named back to `VALID_CLAIMS`